src/components/TypedText.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { useEffect } from "react";
 import { TransitionGroup, CSSTransition } from "react-transition-group";
 import GoogleAnalytics from "react-ga4";
-import { UserSettings } from "../types";
+import { CurrentLessonStrokes, UserSettings } from "../types";
 
 type Props = {
   actualText: string;
   completedPhrases: any;
-  currentLessonStrokes: any;
+  currentLessonStrokes: CurrentLessonStrokes[];
```

Comment:
`currentLessonStrokes` had type mismatch, so I eagerly typed it. [The usage](https://github.com/didoesdigital/typey-type/pull/169/commits/031b3b2c1bab54d5a5ac884dc2ce9ad690f27397#diff-8455b849ac52a2a72f0eb6511c7da55e5ad341e3e2c717c2b64758d4037340e5R81) assumes array. Ancestor components incorrectly typed it `CurrentLessonStrokes` before